Documentation/compatibility/wpf-PackageDigitalSignatureManager.DefaultHashAlgorithm-is-now-SHA256.md

```diff
@@ -0,0 +1,41 @@
+## The default hash algorithm for WPF PackageDigitalSignatureManager is now SHA256
+
+### Scope
+Edge Case
+
+### Version Introduced
+.NET 4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+The PackageDigitalSignatureManager provides functionality for digital signatures in relation to WPF packages.  Previously, the default algorithm (PackageDigitalSignatureManager.DefaultHashAlgorithm)
```

Comment:
PackageDigitalSignatureManager should be a link --> \`System.IO.Packaging.PackageDigitalSignatureManager`
Previously --> In the .NET Framework 4.7 and earlier versions 
DefaultHashAlgorithm should be a link with fully qualified link text --> \<xref:System.IO.Packaging.PackageDigitalSignatureManager.DefaultHashAlgorithm?displayProperty=nameWithType>